driver/js/src/test/esprojects/.interfaces/mlscript/Self.mlsi

```diff
@@ -0,0 +1,4 @@
+declare module Self() {
+  class Foo()
+}
+//| can not import driver/js/src/test/esprojects/mlscript/Self.mls itself
```

Comment:
```suggestion
//| Cannot import driver/js/src/test/esprojects/mlscript/Self.mls itself
```
Does `itself` mean this is a cyclic import? It should be rephrased because currently it's unclear.

Reply:
Yes. In `Self.mls`:
```ts
import "./Self.mls"
```

Reply:
```suggestion
//| Cannot import driver/js/src/test/esprojects/mlscript/Self.mls from itself
```